app/org/sagebionetworks/bridge/services/HealthCodeServiceImpl.java

```diff
@@ -43,15 +44,43 @@ public String getCode() {
     }
 
     @Override
-    public String getHealthCode(String id) {
-        return healthIdDao.getCode(id);
+    public HealthId getMapping(String healthId) {
+        checkNotNull(healthId);
+        final String healthCode = healthIdDao.getCode(healthId);
+        if (healthCode == null) {
+            return null;
+        }
```

Comment:
[POLISH] Would it be worthwhile creating a mock test to sanity test that even if healthIdDao returns a null healthcode (which could also happen if we call with an invalid health ID), that we return null (instead of a mapping with health ID and no code)?